webcompat/views.py

```diff
@@ -176,9 +176,10 @@ def create_issue():
         # HTML <form>, so we stick them in the session cookie so they survive
         # the scenario where the user decides to do authentication, and they
         # can then be passed on to form.py
-        for param in ['src', 'label']:
-            if request.args.get(param):
-                session[param] = request.args.get(param)
+        if request.args.get('src'):
+            session['src'] = request.args.get('src')
+        if request.args.get('label'):
+            session['src'] = request.args.getlist('label')
```

Comment:
I'm confused here. Could you explain?

Reply:
> I'm confused here. Could you explain?

@karlcow oh, just a typo. should be `session['label']` (i didn't catch that because this code is only exercised when doing auth reporting).